autoload/lexima/template_rules.vim

```diff
@@ -20,236 +20,307 @@
 " OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 function! lexima#template_rules#add_rules()
-    augroup lexima_template_rules
-        autocmd!
-    augroup END
+  augroup lexima_template_rules
+    autocmd!
+  augroup END
 
-    " basic html/xml tag delimiters {{{
-    call lexima#add_rule({
+  " basic html/xml tag delimiters {{{
+  call lexima#add_rule({
         \ 'char': '<',
         \ 'input_after': '>',
-        \ 'filetype': ['html', 'jinja', 'htmljinja', 'django', 'htmldjango', 'liquid', 'twig', 'mako', 'xml'],
+        \ 'filetype': ['html', 'jinja', 'htmljinja', 'django', 'htmldjango', 'liquid', 'twig', 'html.twig', 'mako', 'xml'],
         \ })
-    call lexima#add_rule({
+  call lexima#add_rule({
         \ 'char': '>',
         \ 'at': '\%#>',
         \ 'leave': 1,
-        \ 'filetype': ['html', 'jinja', 'htmljinja', 'django', 'htmldjango', 'liquid', 'twig', 'mako', 'xml'],
+        \ 'filetype': ['html', 'jinja', 'htmljinja', 'django', 'htmldjango', 'liquid', 'twig', 'html.twig', 'mako', 'xml'],
         \ })
     call lexima#add_rule({
         \ 'char': '<BS>',
         \ 'at': '<.*>\%#',
         \ 'input': '<BS>',
         \ 'input_after': '>',
-        \ 'filetype': ['html', 'jinja', 'htmljinja', 'django', 'htmldjango', 'liquid', 'twig', 'mako', 'xml'],
+        \ 'filetype': ['html', 'jinja', 'htmljinja', 'django', 'htmldjango', 'liquid', 'twig', 'html.twig', 'mako', 'xml'],
         \ })
     call lexima#add_rule({
```

Comment:
Presumably this line (and 39) being extra indented are the "two minor errors" you mentioned. Otherwise this LGTM. 👍